flow/util/docker_shell

```diff
@@ -26,12 +29,14 @@ docker run -u $(id -u ${USER}):$(id -g ${USER}) \
  -e XAUTHORITY=$XAUTH \
  -e FLOW_HOME=/OpenROAD-flow-scripts/flow/ \
  -e MAKEFILES=/OpenROAD-flow-scripts/flow/Makefile \
- -v $WORKSPACE:`pwd` \
+ -e YOSYS_CMD=$YOSYS_CMD \
+ -e OPENROAD_EXE=$OPENROAD_EXE \
+ -e KLAYOUT_CMD=$KLAYOUT_CMD \
```

Comment:
@luarss, could you remind me why we need to pass these variables to docker?

Reply:
This command sets the env variables if the user specifies it. Otherwise it sets to the default path it would have been inside the docker.